GooglePlayInstant/Editor/PlayInstantPublisher.cs

```diff
@@ -48,16 +48,27 @@ public static void Build()
                 return;
             }
 
+            ZipApk(baseApkPath, zipFilePath);
+        }
+
+        /// <summary>
+        /// Creates a ZIP file containing the specified APK
```

Comment:
Creates a ZIP file containing the specified file in the specified directory.

(Note trailing "." for consistency.)

Reply:
Resolved